applications/crossbar/src/modules/cb_service_plans.erl

```diff
@@ -160,7 +166,29 @@ validate_service_plan(Context, PlanId, ?HTTP_DELETE) ->
 %% (after a merge perhaps).
 %% @end
 %%--------------------------------------------------------------------
+-spec post(cb_context:context()) -> cb_context:context().
 -spec post(cb_context:context(), path_token()) -> cb_context:context().
+post(Context) ->
+    Plans = cb_context:fetch(Context, <<"plans">>, []),
+    Services =
+        lists:foldl(
+            fun(PlanId, Acc) ->
```

Comment:
This should be a named function like `apply_plan(PlanId, Acc)` so we can add type/spec info